app/controllers/cohort_helper.rb

```diff
@@ -5,6 +5,6 @@ def current_cohort
   end
 
   def all_cohorts
-    [2016, 2015, 2014, 2013]
+    [2016, 2015]
```

Comment:
Initially I did not see any data for 2014 and 2013 so I thought those 2 weren't necessary. Will insert 2013 and 2014 back, together with 2017